client/my-sites/themes/multi-site.jsx

```diff
@@ -9,12 +9,17 @@ import React from 'react';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import ThemesSiteSelectorModal from './themes-site-selector-modal';
 import { connectOptions } from './theme-options';
+import QueryUserPurchases from 'components/data/query-user-purchases';
 import ThemeShowcase from './theme-showcase';
+import userFactory from 'lib/user';
+
+const user = userFactory();
```

Comment:
Can we use the `getCurrentUserId` selector from `state/current-user/selectors` instead? `lib/user` is a pre-Redux, `EventEmitter` based lib that we're trying to get rid of.